internal/util/rocksmq/server/rocksmq/rocksmq_impl.go

```diff
@@ -273,48 +273,38 @@ func (rmq *rocksmq) CreateTopic(topicName string) error {
 // DestroyTopic removes messages for topic in rocksdb
 func (rmq *rocksmq) DestroyTopic(topicName string) error {
 	start := time.Now()
-	beginKey := topicName + "/begin_id"
-	endKey := topicName + "/end_id"
-
-	err := rmq.kv.Remove(beginKey)
-	if err != nil {
-		log.Debug("RocksMQ: failed to remove key <" + beginKey + ">.")
-		return err
+	ll, ok := topicMu.Load(topicName)
+	if !ok {
+		return fmt.Errorf("topic name = %s not exist", topicName)
 	}
-
-	err = rmq.kv.Remove(endKey)
-	if err != nil {
-		log.Debug("RocksMQ: failed to remove key <" + endKey + ">.")
-		return err
+	lock, ok := ll.(*sync.Mutex)
+	if !ok {
+		return fmt.Errorf("get mutex failed, topic name = %s", topicName)
 	}
+	lock.Lock()
+	defer lock.Unlock()
+	beginKey := topicName + "/begin_id"
+	endKey := topicName + "/end_id"
+	removedKeys := make([]string, 0)
```

Comment:
It's suggested that to use `var removedKeys []string` instead of zero length make. Please update this kind of usage in next PR.